src/pages/workspaces/workspace/analysis/modals/compute-modal-components.ts

```diff
@@ -0,0 +1,71 @@
+import { div, h, label } from 'react-hyperscript-helpers'
+
+import { Link, Select, useUniqueId } from '../../../../../components/common'
+import { NumberInput } from '../../../../../components/input'
+import TooltipTrigger from '../../../../../components/TooltipTrigger'
+import { pdTypes } from '../../../../../pages/workspaces/workspace/analysis/runtime-utils'
+import { computeStyles } from './modalStyles'
```

Comment:
We should do absolute imports.

My IDE does this too... There's probably a setting for this.